lib/queue_publisher.rb

```diff
@@ -62,6 +64,29 @@ def publish_message(routing_key, message_data, options = {})
     raise
   end
 
+  def validate_message(message_data)
+    schema = JSON.load(File.read("govuk-content-schemas/message_queue.json"))
```

Comment:
Is this validation different to the SchemaValidator? 
https://github.com/alphagov/publishing-api/blob/master/app/validators/schema_validator.rb